tree/ntuple/v7/src/RField.cxx

```diff
@@ -1326,7 +1345,9 @@ void ROOT::Experimental::RCollectionClassField::ReadGlobalImpl(NTupleSize_t glob
 const ROOT::Experimental::Detail::RFieldBase::RColumnRepresentations &
 ROOT::Experimental::RCollectionClassField::GetColumnRepresentations() const
 {
-   static RColumnRepresentations representations({{EColumnType::kSplitIndex32}, {EColumnType::kIndex32}}, {{}});
+   static RColumnRepresentations representations(
```

Comment:
(side question as this is pre-existing)  Can you remind what the advantage of duplicating the `representations` declarations?  And more importantly, does it matter if there internal ordering gets out of sync?

Reply:
Not sure I understand. What do you mean by "duplicating"?